playbooks/provisioning/openstack/sample-inventory/group_vars/all.yml

```diff
@@ -12,6 +12,8 @@ openstack_num_masters: 1
 openstack_num_infra: 1
 openstack_num_nodes: 2
 
+openstack_node_removal_policies: []
```

Comment:
Can you put a short comment here describing what is this for?

Also, with the `default` fix in the provision-openstack playbook, this can be commented out (so please do that).